ArdusatSDK.cpp

```diff
@@ -1,35 +1,52 @@
 /**
  * @file   ArdusatSDK.cpp
- * @Author Ben Peters (ben@ardusat.com)
+ * @author Ben Peters (ben@ardusat.com)
+ * @author Sam Olds (sam@ardusat.com)
  * @date   December 3, 2014
  * @brief  Implements ArdusatSDK generic sensor reading and configuration for Space Kit Sensors.
+ *
+ * Provides a unifying wrapper of sensor specific functionality to provide a
+ * consistent interface to interact with each type of sensor.
  */
 
 #include <stdio.h>
 #include <string.h>
 #include "ArdusatSDK.h"
 
-bool ARDUSAT_SHIELD = false;
-int OUTPUT_BUF_SIZE = 256;
+boolean MANUAL_CONFIG = false;
+boolean ARDUSAT_SPACEBOARD = false;
+int OUTPUT_BUF_SIZE = 512;
```

Comment:
For the `all_sensors` sketch, there was 579 bytes of free ram when `OUTPUT_BUF_SIZE = 256`. Now it's at 323 bytes.

This was a premature change for the CSV Writer stuff. I figured that 256 wasn't a big enough buffer for when someone wants to log all sensor values and it needs to build the initial header row with sensor names and units and stuff.